src/price_data.py

```diff
@@ -627,15 +639,15 @@ def get_price(
             return decimal.Decimal("1")
 
         db_path = self.get_db_path(platform)
-        tablename = self.get_tablename(coin, reference_coin)
+        coin_a, coin_b, reciprocal = self._sort_pair(coin, reference_coin)
+        tablename = self.get_tablename(coin_a, coin_b)
 
         # Check if price exists already in our database.
         if (price := self.__get_price_db(db_path, tablename, utc_time)) is None:
             try:
                 get_price = getattr(self, f"_get_price_{platform}")
             except AttributeError:
                 raise NotImplementedError("Unable to read data from %s", platform)
-
             price = get_price(coin, utc_time, reference_coin, **kwargs)
             assert isinstance(price, decimal.Decimal)
             self.__set_price_db(db_path, tablename, utc_time, price)
```

Comment:
Is this missing here?
```
if reciprocal:
    price = misc.reciprocal(price)
```
Since coin/reference_coin are used for fetching the price from the API, the price still needs to be inverted if `coin_a/b` are flipped.

With this change, it should also work if we keep `buytr.change / selltr.change` (for my comment above for `book.py`). I think this is exactly why there is an inconsistency between prices fetched from CSV and from the APIs.

Reply:
Looks like it would.